securedrop-protocol/src/keys/journalist.rs

```diff
@@ -228,6 +228,25 @@ pub struct JournalistOneTimeKeyBundle {
     pub signature: Signature,
 }
 
+/// One-time keystore (public and private) for a journalist
+/// TODO: improve/refactor with OneTimeKeyBundle
+/// TODO: use native hpke-rs types
+#[derive(Debug, Clone)]
+pub struct JournalistOneTimeKeystore {
+    // One-time PQ KEM PSK (message enc)
+    pub(crate) one_time_message_pq_sk: MLKEM768PrivateKey,
+    pubone_time_message_pq_pk: MLKEM768PublicKey,
+    /// One-time DH-AKEM public key (message enc)
+    pub(crate) one_time_message_sk: DhAkemPrivateKey,
+    pub one_time_message_pk: DhAkemPublicKey,
+    /// One-time XWING public key (metadata enc)
+    pub(crate) one_time_metadata_sk: XWingPrivateKey,
+    pub one_time_metadata_pk: XWingPublicKey,
```

Comment:
ah I just saw we had this defined, we could use `JournalistOneTimeMessagePQKeyPair, JournalistOneTimeMessageClassicalKeyPair, JournalistOneTimeMetadataKeyPair` types here and then just use their struct fields directly since the public keys are `pub` and the private keys are `pub(crate)`